CI/travis.osx.after_success.sh

```diff
@@ -13,7 +13,7 @@ if [ "${DEPLOY}" = "deploy" ]; then
   commitDate=$(git show -s --format=%cs | tr -d '-')
   yesterdaysDate=$(date -v-1d '+%F' | tr -d '-')
 
-  git clone https://github.com/Mudlet/installers.git "${TRAVIS_BUILD_DIR}/../installers"
```

Comment:
This will be removed upon approval.

Reply:
So? 😕 

Don't you need the installer repo stuff - or is it duplicated?

💡 Ah, you mean there is a corresponding change in the installer repository and you have to use a different branch until you get the fixes into BOTH repositories! Yes?

Reply:
Yep